team/bundles/org.eclipse.team.ui/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.team.ui; singleton:=true
-Bundle-Version: 3.10.500.qualifier
+Bundle-Version: 3.10.600.qualifier
```

Comment:
CI-Build says:

The *minor* version should be incremented in version 3.10.600, since new APIs have been added since version 3.10.500

So I think 3.11.000 would be the correct new version not. 3.10.600.